proto/spaceone/api/inventory_v2/v1/region.proto

```diff
@@ -89,22 +97,31 @@ message RegionQuery {
     // +optional
     string name = 3;
     // +optional
-    string region_key = 4;
-    // +optional
     string region_code = 5;
```

Comment:
`region_code` field number need to start from `4` and then exists_only end with `6`